modules/rosa-adding-instance-types.adoc

```diff
@@ -0,0 +1,50 @@
+:_module-type: PROCEDURE
+//Specify the system-module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// * assemblies/rosa-managing-worker-nodes.adoc
+
+[id="rosa-adding-instance-types_{context}"]
+= Adding instance types
+
+[role="_abstract"]
+After a machine pool is created, the instance type cannot be changed. To add a different instance type for worker nodes, you must create a new machine pool for the additional instance type.
+
+.Procedure
+
+. To add an instance type with a new machine pool, enter the following command:
++
+[source,terminal]
+----
+$ rosa create machinepool --cluster=<cluster-name> <machinepool_ID> --instance-type=<type>
+----
++
+.Example input
++
+This example creates a new machine pool with the instance type `m5.2x.large`, 2 replicas, and labels on a cluster named `mycluster`:
++
+[source,terminal]
+----
+$ rosa create machinepool --cluster=mycluster db-nodes-large-mp --replicas=2 --labels=app=db,tier=backend --instance-type=m5.2xlarge
+----
++
+[NOTE]
+====
+Some of the supported AWS compute instance types include: `M5.xlarge`, `M5.2xlarge`, and `M5.4xlarge`. For a complete list of supported instance types, see the _Policies and Service Definition_.
```

Comment:
drop the instance type examples here and simply link out to the service definition here.

Reply:
We can't link out until after it's published - then we can hardcode the link in at that time. But, I did add a link in the Resources section of this assembly for now, and removed the examples. PV2 will eventually link to another title without having to hardcode it.